app/assets/javascripts/liking.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function(){
+  $('.helpful').on('submit',function(event){
```

Comment:
If you have turbolinks enabled, document.ready will not fire except on the first page load. Be careful how you mix document.ready and rails's page:load. My recommendation is to disable turbolinks and always use document ready.

http://blog.steveklabnik.com/posts/2013-06-25-removing-turbolinks-from-rails-4